core/src/com/unciv/ui/options/DisplayTab.kt

```diff
@@ -94,6 +98,56 @@ private fun addMinimapSizeSlider(table: Table, settings: GameSettings, selectBox
     table.add(minimapSlider).minWidth(selectBoxMinWidth).pad(10f).row()
 }
 
+private fun addUnitIconAlphaSlider(table: Table, settings: GameSettings, selectBoxMinWidth: Float) {
+    table.add("Unit icon opacity".toLabel()).left().fillX()
+
+    val offTranslated = "off".tr()
+    val onTranslated = "opaque".tr()
+    val getTipText: (Float) -> String = {
+        when (it) {
+            0f -> offTranslated
+            in 0.01f..0.99f -> "%.0f".format(it*100) + "%"
+            else -> onTranslated
+        }
+    }
+    val unitIconAlphaSlider = UncivSlider(
+        0f, 1f, 0.1f, initial = settings.unitIconOpacity, getTipText = getTipText
+    ) {
+        settings.unitIconOpacity = it
+        settings.save()
+
+        val worldScreen = UncivGame.Current.getWorldScreenIfActive()
+        if (worldScreen != null)
+            worldScreen.shouldUpdate = true
+
+    }
+    table.add(unitIconAlphaSlider).minWidth(selectBoxMinWidth).pad(10f).row()
+}
+
+private fun addUnitIconSizeSlider(table: Table, settings: GameSettings, selectBoxMinWidth: Float) {
+    table.add("Unit icon size".toLabel()).left().fillX()
+
+    val minTranslated = "minimum".tr()
+    val maxTranslated = "maximum".tr()
+    val getTipText: (Float) -> String = {
+        when (it) {
+            10f -> minTranslated
+            in 11f..24f -> "%.0f".format(it)
+            else -> maxTranslated
+        }
+    }
```

Comment:
Probably don't need the text to pop up since dragging the slider to one side will intuitively set it to its minimum or maximum.

Reply:
Yep, I updated it to go from 40% to 100% in 5% increments.